### PR TITLE
refactor(behavior_path_planner): remove unused drivable area parameters

### PR DESCRIPTION
**Pull request by maxime-clem**
## Description

Remove unused parameters

## Tests performed

Not applicable.

## Effects on system behavior

Not applicable.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [ ] I've confirmed the [contribution guidelines].
- [ ] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Review by kosuke55 — APPROVED** [review]

**Comment by xmfcx** [comment]
I've rebased it to the latest main, hopefully all the checks will run this time.

File: planning/behavior_path_avoidance_module/config/avoidance.param.yaml [hunk 1]
```diff
@@ -4,9 +4,7 @@
     avoidance:
       resample_interval_for_planning: 0.3               # [m]
       resample_interval_for_output: 4.0                 # [m]
-      drivable_area_right_bound_offset: 0.0             # [m]
-      drivable_area_left_bound_offset: 0.0              # [m]
-
+
```
Comment (xmfcx) [inline planning/behavior_path_avoidance_module/config/avoidance.param.yaml:7]:
```suggestion

```

Comment (xmfcx) [reply planning/behavior_path_avoidance_module/config/avoidance.param.yaml:7]: removes the trailing spaces for the ci to pass

**Review by xmfcx — CHANGES_REQUESTED** [review]

**Comment by maxime-clem** [comment]
@xmfcx thank you for fixing the CI. Can you approve the PR ? It is blocked by your "request for change".

**Review by xmfcx — APPROVED** [review]
Sorry, sure!

**Review by takayuki5168 — APPROVED** [review]

**Pull request merged**